chef_master/source/resource_template.rst

```diff
@@ -553,7 +553,7 @@ and a resource that looks something like the following:
      group 'root'
    end
 
-This resource would be matched in the same order as the ``/templates`` directory structure. For a node named ``host-node-desktop`` that is running Windows 7, the second item would be the matching item and the location:
+This resource would be matched in the same order as the ``/templates`` directory structure. For a node named ``host-node-desktop`` that is running Windows 8, the second item would be the matching item and the location:
```

Comment:
Without modifying the actual example here this isn't valid anymore. The example directory bit needs an update as well.

Reply:
I updated the examples.

Reply:
Windows 8 here should be 8.1 now or Server 2012R2

Reply:
Fixed.